installer/scripts/omsadmin.sh

```diff
@@ -862,6 +862,7 @@ make_dir()
     fi
 
     chown_omsagent $1
+    chmod 755 $1
```

Comment:
Is this chmod expected to open the permissions, granting access to a wider audience, or closing access (such as removing world write access)? If it is the later, this means there **is** a window of time where unauthorized actors have access they should not have.

Recommendations:

- Look at setting the process umask to something narrow, such as 0027.
- Check that permission granted at file creation time are no more open then they must be, such as 0660.